canonical-websites/www.ubuntu.com
# New finserv month landing page

## Done

* New landing page for financial services month

## TODO before merging

- [x] UTMs
- [x] Copy doc: https://docs.google.com/document/d/1THWC0QzAzGOoaLXWrxil100SkpvXm3tJd9fUlwRZGzg/edit#heading=h.teujcix8oibf

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Go to http://0.0.0.0:8001/engage/financial-services-month and ensure the page looks good
